### PR TITLE
Add landing page content

### PR DESCRIPTION
**Pull request by magmat88**

### CODE REVIEW

**Comment by magmat88** [comment]
Also adjusted padding of loading indicator in this pull request.

File: src/app/styles/variables/colors.scss [hunk 1]
```diff
@@ -6,6 +6,7 @@ $color_contrast: #EE9B00;
 $color_dark-contrast: #BB3E03;
 $color_dark: #001219;
 $color_light: #FBF8EF;
+//unused
```
Comment (earendillo) [inline src/app/styles/variables/colors.scss:9]: if unused then why to keep this?

File: src/app/app.tsx [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 import { Footer } from './components/footer/footer.component';
 import { MainContent } from './components/mainContent/mainContent.component';
-import { Blogs } from './components/blogs/blogs.component';
-import { AboutPage } from './components/aboutPage/aboutPage.component';
-import './app.scss';
+import { MainHeader } from './components/header/header.component';
+import { Reports } from './components/reports/reports.component';
+import { Router, View } from 'react-navi';
+import { StoreContext } from 'storeon/react';
+import React from 'react';
```
Comment (earendillo) [inline src/app/app.tsx:13]: React import should be at the very top. Please rearrange imports to be in logical order

**Review by earendillo — CHANGES_REQUESTED** [review]

File: src/app/styles/variables/colors.scss (hunk 1 above)
Comment (magmat88) [reply src/app/styles/variables/colors.scss:9]: removed

File: src/app/app.tsx (hunk 2 above)
Comment (magmat88) [reply src/app/app.tsx:13]: Imports sorted using standard convention - first import of the React package, then imports of the third-party libraries and in the end imports of the local files. All grouped separately. Applied to all tsx files.

File: src/app/components/mainContent/mainContent.component.tsx [hunk 3]
```diff
@@ -19,28 +22,32 @@ export function MainContent(): JSX.Element {
...
-                />
-            </div>
-            <div className="category-items">
-                <CategoryItems
-                    categoryItems={content.blogs?.slice(0, 2)}
-                    label={ContentLabel.Blog}
-                    length={120}
-                />
+        <Fragment>
+            <LandingPage />
```
Comment (earendillo) [inline src/app/components/mainContent/mainContent.component.tsx:26]: it should be reversed, MainContent should be part of LandingPage, but leave it as it is for now

**Review by earendillo — APPROVED** [review]

**Pull request merged**